### PR TITLE
Add variables to default config

### PR DESCRIPTION
**Pull request by PolskiGlizda**
#### Describe your PR, what does it fix/add?
This PR adds variables for programs that user can exec with default binds. It allows users to easier change the default behavior of binds and  makes default config more readable for new users

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
No

#### Is it ready for merging, or does it need work?
I guess so

### CODE REVIEW

File: src/config/defaultConfig.hpp [hunk 1]
```diff
@@ -28,6 +28,12 @@ monitor=,preferred,auto,auto
 # Source a file (multi-file configs)
 # source = ~/.config/hypr/myColors.conf
 
+# Set programs that you use
+$terminal = 'kitty'
+$FileManager = 'dolphin'
+$Menu = 'wofi --show drun'
```
[the remark this answers is not in the document]
Comment (PolskiGlizda) [reply src/config/defaultConfig.hpp]: oh sorry it's a typo I meant for all of it to be in camelCase

[the remark this answers is not in the document]
Comment (PolskiGlizda) [reply src/config/defaultConfig.hpp]: I fixed it

**Comment by Eclextic** [comment]
> just because your shell sanitizes those doesn't mean every single one will, besides, it's confusing.

Variables in Hyprland work anyways in a line-to-line basis, meaning no quotes, no nothing.
See envvar as they work the same way:
https://wiki.hyprland.org/Configuring/Environment-variables/

`Hyprland puts the raw string to the envvar with the env keyword. You should not add quotes around the values.`

File: src/config/defaultConfig.hpp [hunk 2]
```diff
@@ -28,6 +28,12 @@ monitor=,preferred,auto,auto
 # Source a file (multi-file configs)
 # source = ~/.config/hypr/myColors.conf
 
+# Set programs that you use
+$terminal = kitty
+$fileManager = dolphin
+$menu = wofi --show drun
+
```
Comment (vaxerski) [inline src/config/defaultConfig.hpp:35]: why the newlines? either remove them or add them to the example config too.

**Review by vaxerski — COMMENTED** [review]
rest lgtm

**Review by vaxerski — APPROVED** [review]
ok

**Pull request merged**